app/views/devise/registrations/edit.html.erb

```diff
@@ -5,20 +5,15 @@
     <%= devise_error_messages! %>
 
     <h3>Change your account details</h3>
-    <%= f.email_field :email, autofocus: true %>
+    <p>Email</p>
+    <p><%= current_user.email %></p>
```

Comment:
Check out this cool thing in our bootstrap library:

https://github.com/bootstrap-ruby/rails-bootstrap-forms#static-controls

I think we can use the static_control :email thing here, and that'll keep the styling similar. Thoughts?